src/platform-implementation-js/lib/fromEventsWithOptions.ts

```diff
@@ -1,20 +1,14 @@
 import * as Kefir from 'kefir';
 
-import {
-  Options,
-  addEventListener,
-  removeEventListener,
-} from 'event-listener-with-options';
-
 export default function fromEventsWithOptions(
   target: EventTarget,
   eventName: string,
-  options: Options | boolean,
+  options: boolean,
```

Comment:
This doesn't match the calling code's shape that's passed in.

```suggestion
  options: Parameters<typeof target.addEventListener>[2],
```